docs/compute-element/covid-19.md

```diff
@@ -41,70 +41,75 @@ system:
 ```
 JOB_ROUTER_ENTRIES @=jre
 [
- name = "OSG Jobs";
+ name = "OSG_Jobs";
  GridResource = "batch slurm";
  TargetUniverse = 9;
  queue = "osg";
 ]
 ```
 
 To add a new route for COVID-19 pilots, append a new route
-prior to the existing one:
+prior to the existing one and specify their order:
 
-```
+```hl_lines="2 3 4 5 6 7 8 17 18"
 JOB_ROUTER_ENTRIES @=jre
 [
- name = "OSG COVID-19 Jobs";
+ name = "OSG_COVID-19_Jobs";
  GridResource = "batch slurm";
  TargetUniverse = 9;
  queue = "covid19";
  Requirements = (TARGET.IsCOVID19 =?= true);
 ]
 [
- name = "OSG Jobs";
+ name = "OSG_Jobs";
  GridResource = "batch slurm";
  TargetUniverse = 9;
  queue = "osg";
 ]
 @jre
+
+# Specify the order of the routes
+JOB_ROUTER_ROUTE_NAMES = OSG_COVID-19_Jobs, OSG_Jobs
 ```
 
 To verify jobs are being routed appropriately,
-[one may examine pilots](compute-element/troubleshoot-htcondor-ce/#condor_ce_router_q)
+[one may examine pilots](/compute-element/troubleshoot-htcondor-ce/#condor_ce_router_q)
 with `condor_ce_router_q`.
 
 !!! note "Additional considerations for older CEs"
-    Prior to HTCondor 8.7.1, HTCondor-CE had separate rules for handling
+    Prior to HTCondor 8.8.8 or 8.9.5, HTCondor-CE had separate rules for handling
```

Comment:
```suggestion
    Prior to HTCondor 8.8.6 or 8.9.5, HTCondor-CE had separate rules for handling
```

Reply:
Oops, `JOB_ROUTER_ROUTE_NAMES` was actually added in 8.8.7: https://htcondor.readthedocs.io/en/stable/version-history/stable-release-series-88.html#version-8-8-7